src/main.ts

```diff
@@ -97,26 +99,33 @@ async function main() {
  * @param wsUrl websocket url of the node Sidecar is connected to
  * @param chainName chain name of the network Sidecar is connected to
  * @param implName implementation name of the node Sidecar is connected to
+ * @param httpUrl http url of the node Sidecar is connected to. This is optional,
+ * but when used will override the ws url.
  */
-function startUpPrompt(wsUrl: string, chainName: string, implName: string) {
+function startUpPrompt(
+	wsUrl: string,
```

Comment:
I reckon that it might be cleaner just to use `url` here and get rid of the ´wsUrl and httpUrl`